lib/fastlane/actions/get_github_release.rb

```diff
@@ -109,6 +110,15 @@ def self.available_options
                                          raise "Please only pass the path, e.g. 'KrauseFx/fastlane'".red if value.include? "github.com"
                                          raise "Please only pass the path, e.g. 'KrauseFx/fastlane'".red if value.split('/').count != 2
                                        end),
+          FastlaneCore::ConfigItem.new(key: :server_url,
+                                       env_name: "FL_GITHUB_RELEASE_SERVER_URL",
+                                       description: "The server url. e.g. 'https://your.github.server/api/v3' (Default: 'https://api.github.com')",
+                                       default_value: "https://api.github.com",
+                                       optional: true,
+                                       verify_block: proc do |value|
+                                         raise "Please don't include the trailing slash in the server url".red if value.end_with? "/"
```

Comment:
Was just more code in more places, but i could do that if you'd prefer

On Tue, Feb 9, 2016, 5:37 PM Sam Phillips <notifications@github.com> wrote:

> In lib/fastlane/actions/get_github_release.rb
> <https://github.com/fastlane/fastlane/pull/1291#discussion_r52404422>:
>
> > @@ -109,6 +110,15 @@ def self.available_options
> >                                           raise "Please only pass the path, e.g. 'KrauseFx/fastlane'".red if value.include? "github.com"
> >                                           raise "Please only pass the path, e.g. 'KrauseFx/fastlane'".red if value.split('/').count != 2
> >                                         end),
> > +          FastlaneCore::ConfigItem.new(key: :server_url,
> > +                                       env_name: "FL_GITHUB_RELEASE_SERVER_URL",
> > +                                       description: "The server url. e.g. 'https://your.github.server/api/v3' (Default: 'https://api.github.com')",
> > +                                       default_value: "https://api.github.com",
> > +                                       optional: true,
> > +                                       verify_block: proc do |value|
> > +                                         raise "Please don't include the trailing slash in the server url".red if value.end_with? "/"
>
> How about about just stripping the extra /?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fastlane/fastlane/pull/1291/files#r52404422>.
>